runner/inventory.go

```diff
@@ -2,23 +2,17 @@ package runner
 
 import (
 	"encoding/json"
-	"fmt"
 	"os"
 	"text/template"
 
 	log "github.com/sirupsen/logrus"
 
-	"github.com/trento-project/trento/internal/cloud"
-	"github.com/trento-project/trento/internal/cluster"
-	"github.com/trento-project/trento/internal/consul"
-	"github.com/trento-project/trento/internal/hosts"
-
 	"github.com/trento-project/trento/api"
 )
 
 type InventoryContent struct {
 	Groups []*Group
-	Nodes  []*Node
+	Nodes  []*Node // this seems unused
```

Comment:
Yes. This attribute is already there for any future circuntance where we would need to run ansible checks in ungrouped nodes